apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -197,16 +197,21 @@ export const EventSetupTab = (
             name={`locations.${index}.${eventLocationType.defaultValueVariable}`}
             control={formMethods.control}
             defaultValue={defaultValue}
-            render={({ field: { onChange, value } }) => {
+            render={({ field: { value } }) => {
               return (
-                <Input
+                <LocationsAutocomplete
```

Comment:
This input is used by various locations like discord, around.co, link meeting ..etc and not just "In Person (Organizer address)" 

We should be making api request for every location input type of text.

Reply:
If I'm not mistaken, places autocomplete should be only for physical (In-Person) input?